packages/amplify-graphql-validate-transformer/src/types.ts

```diff
@@ -0,0 +1,26 @@
+/* c8 ignore start */
+import { ObjectTypeDefinitionNode, FieldDefinitionNode } from 'graphql';
+/* c8 ignore end */
+
+// All supported validation types
```

Comment:
Here and throughout: since you're exporting these symbols, use JSDoc style comments (`/** ... */`) so they appear in IDE documentation